script.js

```diff
@@ -0,0 +1,10 @@
+function sum() {
+    var a,b;
+        var a =Number(document.getElementById("a").value)
+        var b =Number(document.getElementById("b").value)
+     sum = a + b 
```

Comment:
You should not use the same name for your variables as you do for your functions. What you did here is create a function called "sum" and then the first time the function runs, it creates a variable called the same thing, overriding your function. Now your function doesn't exist. The next time you try to use the calculator, "sum" is no longer a function so it causes an error.